scripts/install.sh

```diff
@@ -39,16 +49,29 @@ fi
 
 echo "$OUTPUT_BINARY installed to /usr/local/bin"
 
-# create directory to hold our data if it does not already exist
+# Create directory to hold our data if it does not already exist
 # /usr/local/share is a good place to store read only data for our program 
 sudo mkdir -p /usr/local/share/pproc
 
-# copy hashes data to data directory 
+# Copy hashes data to data directory with proper permissions
 sudo cp "$PROJECT_ROOT/hashes/sha1-hashes.txt" /usr/local/share/pproc/sha1-hashes.txt
 echo "sha1 hashes copied to /usr/local/share/pproc/sha1-hashes.txt"
 sudo cp "$PROJECT_ROOT/hashes/sha256-hashes.txt" /usr/local/share/pproc/sha256-hashes.txt
 echo "sha256 hashes copied to /usr/local/share/pproc/sha256-hashes.txt"
 sudo cp "$PROJECT_ROOT/hashes/md5-hashes.txt" /usr/local/share/pproc/md5-hashes.txt
 echo "md5 hashes copied to /usr/local/share/pproc/md5-hashes.txt"
 
+# Set proper permissions for the hash files
+sudo chmod 644 /usr/local/share/pproc/*.txt
+
+echo "Hash files copied to /usr/local/share/pproc/"
+
+# Create home directory log file for non-root usage
+if [ -n "$SUDO_USER" ]; then
```

Comment:
You should always be sudo user when using install script check may be unnecessary